Vic2ToHoI4/Data_Files/blankMod/output/localisation/autonomy_l_english.yml

```diff
@@ -0,0 +1,65 @@
+﻿l_english:
+ autonomy_sphereling:0 "Sphereling"
```

Comment:
Better to put this in a new file (autonomy_converted_l_english.yml?), so we don't have to keep everything else up to date as HoI4 changes.

Reply:
The rest looks fine